pkg/shark/wiresharkcfg/parser_test.go

```diff
@@ -2291,7 +2291,7 @@ gui.qt.font_name: Liberation Mono,11,-1,5,50,0,0,0,0,0
 
 # Whether the dissector should print items of h245 Info column in reversed order
 # TRUE or FALSE (case-insensitive)
-#h245.prepand: FALSE
+#h245.prepend: FALSE
```

Comment:
Please double-check this

Reply:
It's in a section of config I stole from Wireshark - safe to apply, thank you!